Jolly.xcodeproj/GeneratedModuleMap/CHTTPParser/module.modulemap

```diff
@@ -0,0 +1,5 @@
+framework module CHTTPParser {
+    umbrella "/Users/pablovillar/Desktop/Projects/Jolly/Packages/CHTTPParser-0.3.0/Sources/CHTTPParser/include"
```

Comment:
I'm new around here, but… isn't it a bad practice to push your local path to github?